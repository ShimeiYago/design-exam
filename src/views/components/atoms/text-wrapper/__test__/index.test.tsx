import { shallow, ShallowWrapper } from 'enzyme';
import { TextWrapper, Props } from '..';

let wrapper: ShallowWrapper<Props>;

const props: Props = {
  children: 'text',
  size: 'medium',
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<TextWrapper {...props} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('align center', () => {
    wrapper.setProps({ align: 'center' });
    expect(wrapper).toMatchSnapshot();
  });

  it('bold case', () => {
    wrapper.setProps({ bold: true });
    expect(wrapper).toMatchSnapshot();
  });
});
