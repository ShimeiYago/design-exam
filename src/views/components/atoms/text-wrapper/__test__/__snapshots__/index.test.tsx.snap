// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests align center 1`] = `
<p
  className="text medium center"
>
  text
</p>
`;

exports[`Shallow Snapshot Tests basic 1`] = `
<p
  className="text medium"
>
  text
</p>
`;

exports[`Shallow Snapshot Tests bold case 1`] = `
<p
  className="text medium bold"
>
  text
</p>
`;

exports[`Shallow Snapshot Tests gray color case 1`] = `
<p
  className="text medium gray"
>
  text
</p>
`;
